packages/EasyCodingStandard/packages/Configuration/src/Configuration.php

```diff
@@ -88,6 +94,11 @@ public function showErrorTable(): bool
         return $this->showErrorTable;
     }
 
+    public function getCacheDirectory(): string
+    {
+        return $this->cacheDirectory ?? sys_get_temp_dir() . DIRECTORY_SEPARATOR . '_changed_files_detector';
```

Comment:
Where is cache directory set to configuration?

`ParameterProvider`  could be used instead